src/Etg.Yams.Client/YamsClient.cs

```diff
@@ -27,21 +29,21 @@ public class YamsClient : IYamsClient
 
         public async Task Connect()
         {
-            Trace.TraceInformation($"Connecting IPC connections..");
+            Trace.TraceInformation($"Connecting IPC connections..".FormatMessage(_options));
             if (_initConnection != null)
             {
-                await _initConnection.Connect().Timeout(_config.ConnectTimeout, "IPC Monitored Initialization connection failed to connect");
-                Trace.TraceInformation("IPC Monitored Initialization connection connected!");
+                await _initConnection.Connect().Timeout(_config.ConnectTimeout, "IPC Monitored Initialization connection failed to connect".FormatMessage(_options));
+                Trace.TraceInformation("IPC Monitored Initialization connection connected!".FormatMessage(_options));
```

Comment:
This could be something like:
```csharp
TraceInformation("IPC Monitored Initialization connection connected!")
```

```csharp
private void TraceInformation(string msg)
{
    Trace.TraceInformation($"[{_options.AppName} ({_options.AppVersion})] {message}");
}
```